src/main/java/io/github/dsheirer/util/StringUtils.java

```diff
@@ -20,6 +20,9 @@
 
 public class StringUtils
```

Comment:
This is a great addition and makes it easy to apply across the codebase.  Thanks!!